### PR TITLE
Add new submission introduction in release-note

### PR DESCRIPTION
**Pull request by debuggy**

### CODE REVIEW

File: RELEASE_NOTE.md [hunk 1]
```diff
@@ -5,7 +5,14 @@
 - Marketplace related update
   - Please see [Marketplace](https://github.com/microsoft/openpaimarketplace/releases/tag/v1.7.0) for more details
 
-- Introduce new job submission page
+- New job submission page
+  - Please refer to [new submission tutorial] for how to use new submission page.
```
Comment (Binyang2014) [inline RELEASE_NOTE.md:9]: [new submission tutorial] add link for this?

File: RELEASE_NOTE.md [hunk 2]
```diff
@@ -5,7 +5,14 @@
...
+  - New submission page replaces `Advanced` with `More info` and places it under each section to improve user experience.
+  - In new submission page, the sidebar can shrink to give the main area more visual space.
```
Comment (Binyang2014) [inline RELEASE_NOTE.md:11]: `can be shrank`?

Comment (debuggy) [reply RELEASE_NOTE.md]: fixed

File: RELEASE_NOTE.md (hunk 1 above)
Comment (debuggy) [reply RELEASE_NOTE.md]: currently the tutorial is in a pull request and will merged later today. So the link might not work right now

**Review by Binyang2014 — APPROVED** [review]

**Pull request merged**